tests/scripts/check-names.sh

```diff
@@ -70,7 +70,7 @@ NL='
 '
 sed -n 's/MBED..._[A-Z0-9_]*/\'"$NL"'&\'"$NL"/gp \
     $HEADERS library/*.c \
-    | grep MBEDTLS | sort -u > _MBEDTLS_XXX
+    | grep MBEDTLS | grep -v MBED_OS_MBEDTLS | sort -u > _MBEDTLS_XXX
```

Comment:
You are right, with the current approach we don't have any better option than this, but it still feels a little bit like cheating. I think this alone is a good reason to not commit the config.h changes into our standalone repository, but generate them with our mbed OS import script.